cmd/access/node_builder/access_node_builder.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/libp2p/go-libp2p/core/host"
 	"github.com/libp2p/go-libp2p/core/routing"
 	"github.com/onflow/flow/protobuf/go/flow/access"
-	"github.com/onflow/go-bitswap"
```

Comment:
I suggest keeing the `onflow` path, maybe a new repo or updating the current one.